pkg/code-generator/docs_gen_test.go

```diff
@@ -47,7 +47,11 @@ var _ = Describe("DocsGen", func() {
 		// Create temp directory and path variables
 		workingDir, err := os.Getwd()
 		Expect(err).NotTo(HaveOccurred())
-		tempDir, err = ioutil.TempDir(workingDir, "doc-gen-test-")
+		outputDir := filepath.Join(workingDir, "_output")
```

Comment:
We previously wrote output files from this test to a `docs-gen-test` directory. I moved this into the `_output` directory so that we can use `make clean` to easily clean these up